Gaijinizer patch/Scripts.txt

```diff
@@ -3100,33 +3099,33 @@ What you are doing 'intervention' should have something to do with it.
  我々は、その時を心待ちにしております。
  =================================================="
 > CONTEXT: Scripts/テキスト出力/212:12
-"Subject: Congratulations
-Sender [Meira]
-
- 《Text》==========================================
- Dr. Kurosawa. Has invented another means of communication than the 'device'
- We will send you a message here.
-
-First of all, congratulations on 'completion of the story'.
- Whatever the outcome, he escaped that city
- The 'help' you can do in the world is over.
-But...I think you feel thin
- Apparently, it seems far from our hope of ending.
-
-If you are not convinced by this end
- If you are prepared to aim for a better end ...
-We are for you to see that world again
-There is a preparation.
-Of course, it is not 'the same world.'
-You will need to choose 'Start from scratch'.
-
-But you are different from the first time ...
-By leading one story to the end
- 'The girls' in my room will surely help you.
-Your experience and mind corruption will be taken over and helpful.
-Please be courageous and step into that world again.
- We look forward to that time.
- =================================================="#MTLed
+"Sender [Meira]
+Subject: Congratulations
+
+ 《Body》==========================================
+As Professor Yui has invented a device that allows for another method
+of communication, I shall send you a message from here.
+
+First of all, congratulations on completing the story. Regardless
+of the outcome, he escaped that city, and the assistance you've
+rendered from your world is now over. However, I'm sure you'll
+be able to appreciate on some level that this does not appear to
+be the conclusion we had hoped for.
+
+Supposing you are not satisfied with this ending, and you're prepared
+to aim for a better conclusion, we are willing to provide another
+opportunity to peek into that world. It won't be the same world,
+of course. And you'll have to choose to start from the beginning.
+
+Having said that, it won't be the same as the first time you started
+from the beginning. Since you've already led one story to its conclusion,
+the girls in "My Room" should be more than willing to lend their strength
+to you.
+
```

Comment:
More flow:
```
Having said that, it won't be exactly like the first time...
Since you've already guided this story to its conclusion, the
girls in "My Room" should be more then willing to lend their
strength to you.
```